rllib/policy/policy.py

```diff
@@ -236,6 +267,37 @@ def set_weights(self, weights):
         """
         raise NotImplementedError
 
+    @DeveloperAPI
+    def get_exploration_state(self):
+        """
+        Returns the current exploration state of this policy, which depends
+        on the policy's Exploration object.
+
+        Returns:
+            any: Serializable copy or view of the current exploration state.
+        """
+        if self.exploration is not None:
+            raise NotImplementedError
+
+    @DeveloperAPI
+    def set_exploration_state(self, exploration_state):
+        """Sets the current exploration state of this Policy.
+
+        Arguments:
+            exploration_state (any): Serializable copy or view of the new
+                exploration state.
+        """
+        if self.exploration is not None:
+            raise NotImplementedError
+
+    @DeveloperAPI
+    def is_recurrent(self):
+        """
```

Comment:
Doc style.

Reply:
It's not written/added by me. Yes, it seems redundant. I might have moved it here (from further down). I'll check. ...

Reply:
done